Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN set -x && \
     # gnupg2:     GPG keys
     # rng-tools:  to generate enough _random_ data for GPG keys
     # rpm-sign:   rpm signing
-    dnf -y install httpd vsftpd python2-behave python2-six python-enum34 python2-whichcraft python-jinja2 python2-pexpect rpm-build openssl mod_ssl gnupg2 rng-tools rpm-sign && \
+    dnf -y install httpd vsftpd python2-behave python2-six python-enum34 python2-whichcraft python-jinja2 python2-pexpect rpm-build openssl mod_ssl gnupg2 rng-tools rpm-sign psmisc && \
```

Comment:
I have uploaded an updated version that doesn't need the new package, the solution was obvious. :-)